interface/usergroup/facilities_add.php

```diff
@@ -208,32 +208,35 @@ function displayAlert()
     <input type=hidden name=mode value="facility">
     <table border=0 cellpadding=0 cellspacing=0>
         <tr>
-        <td><span class="text"><?php echo xlt('Name'); ?>: </span></td><td><input type=entry name=facility size=20 value=""><span class="mandatory">&nbsp;*</span></td>
+        <td width='150px'><span class="text"><?php echo xlt('Name'); ?>: </span></td><td width='220px'><input type=entry name=facility size=20 value=""><span class="mandatory">&nbsp;*</span></td>
```

Comment:
This is OK for now, but CSS should be used not hard coded style and fix pixels fail on mobile devices.